### DISCUSSION TITLE
vite react app blank page

### DISCUSSION THREAD

**Discussion by Bey1222** [community/community#52695] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hi I am new to react I have successfully deployed my web app but when I open the web app all I see is the background colour I chose, here is my repo https://github.com/Bey1222/lexandjem and the deployed app https://bey1222.github.io/lexandjem/ pls help this has been on for 5 days now

**Comment by michaelwang07** [comment]
I have the same exact problem of having a blank page. I can see the html file when Inspect the page.  Trying to find a solution https://michaelwang07.github.io/gh_pages_test2/

**Comment by mxn2020** [comment]
I just started to use vite today and all i get is a blank page as soon as I import an array or anything else.

**Comment by SRayen** [comment]
[Bey1222](https://github.com/Bey1222)
In vite.config.js : try to update base like that :     base:'/'
It worked !
![git-dissc](https://github.com/community/community/assets/13922445/c2765a2f-09f1-4526-ae75-d1d4d0c1cda2)

**Reply by GhazouaniHichem** [reply]
Great job SRayen.
Working perfectly, thank you.

**Reply by fredjuma007** [reply]
only works on localhost. am using render and am still getting a blank page

**Reply by hrithikapps** [reply]
Working now. Thanks .

**Reply by Toluwani2000** [reply]
i am also having the same issue, I have used my base"/" b ut it is still showing a blank page but it is working locally https://github.com/Toluwani2000/vitejs-landingPage

**Reply by abhisheksharmayt** [reply]
@Toluwani2000 I checked your deployed project and it is working fine what did you do to make it work?
Here's mine https://github.com/abhisheksharmayt/portfolio/tree/vite_ts_setup
I am migrating it from create-react-app to vite but I am getting a blank page.

**Reply by karimovie** [reply]
It Works, Thanks !

**Comment by Vignesh12334** [comment]
I'm building a project using vite and React, while building my project, it suddenly became a white screen and showed this in my terminal

Files in the public directory are served at the root path.
Instead of /public/assets/images/apple.svg, use /assets/images/apple.svg.
